client/packages/common/src/ui/layout/tables/material-react-table/MaterialReactTable.tsx

```diff
@@ -15,14 +15,20 @@ import { Box } from '@openmsupply-client/common';
 
 interface TableProps<T extends MRT_RowData> {
   table: MRT_TableInstance<T>;
+  forceFullWidth?: boolean;
 }
 
 export const MaterialTable = <T extends MRT_RowData>({
   table,
+  forceFullWidth,
 }: TableProps<T>) => {
-  return (
-    <Box sx={{ width: '100%' }}>
-      <MaterialReactTable table={table} />
-    </Box>
-  );
+  if (forceFullWidth) {
+    return (
+      <Box sx={{ width: '100%' }}>
+        <MaterialReactTable table={table} />
+      </Box>
+    );
+  }
```

Comment:
It'd be nice to find a way around this, but I haven't been able to set `width: 100%` on the MaterialTable component's outermost div. 

The reason I've made it conditional here is that, without it, tables with fewer columns don't fill the full available horizontal space. But when this is on, the "Detail" table (with row virtualisation) ends up with either a double-vertical scroll, or single vertical scroll around the wrong div. 

I don't think this is right yet, but we should probably re-address once all the functionality is in.